### DISCUSSION TITLE
Aetna Data Missing

### DISCUSSION THREAD

**Discussion by lmiller251** [CMSgov/price-transparency-guide#733] [Q&A] [answered]
Hello All,

Has anyone else had issues pulling Aetna allowed amount in the past few months? It seems that all 100k+ files are blank. I find it hard to believe they have no reportable data. If anyone has insight into this I would be very grateful. 

Thanks.

**Comment by JR-CMS** [comment, marked as answer]
If you believe you are witnessing an instance of non-compliance with the Transparency in Coverage final rules, please submit a complaint to us here: https://www.cms.gov/healthplan-price-transparency/contact-us

Thank you

**Comment by knash05** [comment]
I saw some with data published on the fully insured link.  I did not find any on the under100 link but didn't expect to see 20 claims on anything but larger groups.  Also not seeing any on the student health link.
